google/cloud/bigquery/v2/minimal/internal/bigquery_http_response.cc

```diff
@@ -24,6 +25,9 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 namespace rest = ::google::cloud::rest_internal;
 
+BigQueryHttpResponse::BigQueryHttpResponse()
+    : http_status_code(static_cast<rest_internal::HttpStatusCode>(0)) {}
```

Comment:
Why not use an actual `HttpStatusCode` value, like `HttpStatusCode::kOk`, instead of casting arbitrary integers?